actions.md

```diff
@@ -126,7 +130,11 @@ These are the primary actions that a user of libaktualizr can perform through th
       - [x] Send an event report (see below)
     - [x] Install an update on the primary
       - [ ] Install an OSTree update on the primary
-      - [ ] Notify "reboot needed" after an OSTree update
+      - [ ] Notify "reboot needed" after an OSTree update trigger
+      - [ ] Set new version to pending status after an OSTree update trigger
+      - [ ] Send EcuInstallationAppliedReport to server after an OSTree update trigger
+      - [ ] Uptane check for updates and manifest sends should be disabled while an installation is pending reboot
+      - [ ] Should trigger a system reboot if `reboot_after_install` is set in the configuration
```

Comment:
Minor, but I've preferred stating things definitively instead of using "should", i.e. "Disable checking for updates and sending manifest while an installation is pending reboot" and "Trigger a system reboot..."